tools/block-sync/Dockerfile

```
@@ -0,0 +1,6 @@
+FROM quay.io/prometheus/busybox:latest
+LABEL maintainer="The Prometheus Authors <prometheus-developers@googlegroups.com>"
+
+COPY block-sync /bin/blocksyncer
```

```suggestion
COPY block-sync /bin/block-sync
```

Let's be consistent with the documentation.